tests/integration/cattletest/core/test_svc_discovery.py

```diff
@@ -2229,6 +2235,46 @@ def test_validate_image(client, context):
     assert e.value.error.fieldName == 'imageUuid'
 
 
+def test_validate_restart_policy(client, context):
+    env = client.create_environment(name=random_str())
+    env = client.wait_success(env)
+    assert env.state == "active"
+
+    restart_policy = {"maximumRetryCount": 0, "name": "no"}
```

Comment:
@sonchang  thats docker thing. Restart policy name can be one of the values below:

--restart="no"             Restart policy (no, on-failure[:max-retry], always)

and for on-failure you can also specify max-retry count param